release/release.gradle

```diff
@@ -244,3 +246,170 @@ artifacts {
 
 task release( dependsOn: [uploadDocumentation, uploadBundlesSourceForge] )
 
+def gitRemote = 'origin'
+if ( project.hasProperty( 'gitRemote' ) ) {
+    gitRemote = project.property( 'gitRemote' )
+}
+
+task updateChangeLogFile {
+    group = "Release"
+    description = "Updates the changelog.txt file and push the changes to github"
+    doFirst{
+        ChangeLogFile.update( ormVersion.fullName );
+
+        executeGitCommand(
+                "git add .",
+                "An error occurred during the execution of git add for the changelog.txt and the release version changes"
+        )
+
+        executeGitCommand(
+                "git commit -m \"${project.ormVersion.fullName}\"",
+                "An error occurred during the execution of git commit for the changelog.txt and the release version changes"
+        )
+
+        executeGitCommand(
+                "git push ${gitRemote}",
+                "An error occurred during the execution git push for the changelog.txt and the release version changes"
+        )
+    }
+}
+
+task ciRelease() {
+    group = "Release"
+    description = "Performs a release: the hibernate version is set and the changelog.txt file updated, the changes are pushed to github, then the release is performed, tagged and the hibernate version is set back to the development one"
```

Comment:
I'd remove "back" as it's not necessarily "going back" to the existing version. More likely to the next version?